core/src/main/java/com/javadiscord/jdi/core/Discord.java

```diff
@@ -152,6 +155,29 @@ public void start() {
         EXECUTOR.execute(discordRequestDispatcher);
     }
 
+    public void stop() {
+        if (this.webSocketManager != null) {
+            this.webSocketManager.stop();
+        }
+
+        discordRequestDispatcher.stop();
+
+        EXECUTOR.shutdown();
+        try {
+            if (!EXECUTOR.awaitTermination(30, TimeUnit.SECONDS)) {
+                EXECUTOR.shutdownNow();
+                if (!EXECUTOR.awaitTermination(30, TimeUnit.SECONDS)) {
+                    LOGGER.error("Threads not terminated properly");
```

Comment:
Can we rename this log message please, maybe something like:

"Executor failed to shutdown within the specified time limit, some tasks may still be running"